### PR TITLE
Update counts to match new data model

### PR DESCRIPTION
**Pull request by pschumm**
Minimal changes required to the portal counts in order to bring up the IBDGC commons with the new data model.

### CODE REVIEW

**Comment by pschumm** [comment]
FYI, I added the missing `.travis.yml` file to https://github.com/uc-cdis/ibdgc-dictionary and the check(s) pass now, so https://s3.amazonaws.com/dictionary-artifacts/ibdgc-dictionary/v1.0.0/schema.json should be reachable. Sorry for my goof.

**Comment by paulineribeyre** [comment]
Hello, the tests are failing on your setup, to fix it you can add this block to your `gitops.json` file, in the `components` `index` block (under the `buttons` block):
```
      "homepageChartNodes": [
        {
          "node": "case",
          "name": "Participants"
        },
        {
          "node": "sample",
          "name": "Samples"
        }
      ]
```

File: ibdgc.datacommons.io/portal/gitops.json [hunk 1]
```diff
@@ -46,6 +41,16 @@
           "label": "Learn more"
         }
       ]
```
Comment (paulineribeyre) [inline ibdgc.datacommons.io/portal/gitops.json]:
```suggestion
      ],
```

Comment (pschumm) [reply ibdgc.datacommons.io/portal/gitops.json]: Ugh—thanks!

**Pull request merged**